custom_components/tuya_local/devices/gratkit_filament_dryer.yaml

```diff
@@ -0,0 +1,199 @@
+name: GratKit filament dryer
+products:
+  - id: mxct11k2jqpak13a
+    name: GratKit Firefly Smart Dryer Box
+primary_entity:
+  entity: switch
+  icon: "mdi:heat-pump"
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+secondary_entities:
+  - entity: number
+    name: Drying temperature
```

Comment:
I wonder if the switch, drying temperature and temperature sensor should be combined into a climate entity, to allow the use of the climate card for control.